content/04.results.md

```diff
@@ -0,0 +1,24 @@
+## Results
+
+### Comparing bioRxiv to PubMed Central
+
+#### Global View
+1. Provide table of top 100 terms that are different between bioRxiv and PubMed Central
+2. Might want to add a word cloud that highlights the difference between both repositories 

```

Comment:
I'd be careful about overusing word clouds. I think your use of them to examine PC axes is probably a better use than here.